### ISSUE TITLE
have open api in shopizer?

### ISSUE THREAD

**Issue by owenludong** [shopizer-ecommerce/shopizer#171]
hi
this is a question,not a bug
I have an ERP system,  I want to adapt shopizer like magento in my ERP system.   so have open api in shopizer project?

for example in magento  :http://devdocs.magento.com/guides/v2.1/rest/bk-rest.html

**Comment by shopizer-ecommerce** [comment]
Yes there i a REST api for managing the catalogue

Can you be more specific on what exactly you are looking for

**Comment by owenludong** [comment]
product api:
1. create a product 
2. get a product
3. update a product
4. list all product

order api:
1.get an order
2.get recently changed orders
3.modify tracking of  a shipped order

have any documents in www.shopizer.com
or  in java source code?

**Comment by shopizer-ecommerce** [comment]
We have just started documenting the API on swagger. We plan to have product somewhere in the next week

https://app.swaggerhub.com/apis/shopizer/Shopizer/1.0.0

**Comment by owenludong** [comment]
the api only adapt shopizer 2.0.6,  I can not find /api/v1/private url in 2.0.5 java source code?

**Comment by shopizer-ecommerce** [comment]
Yes the documentation aligns with the upcoming version 2.2.0 that will be released on Sept 24 th

**Comment by owenludong** [comment]
Thanks very much, last question:   the api is compatible with 2.0.5 version

**Comment by loamDeLest** [comment]
I thought that version 2.2.0 would be released on Sept 24th ?

**Comment by owenludong** [comment]
@shopizer-ecommerce  plan have changed?

**Comment by Harikumar200** [comment]
hi what is the API address ?
I want to test in POSTMAN. What is the url I should use. I deployed app in localhost:8080.

**Comment by sateesh2020** [comment]
@Harikumar200 
one you run your sprint boot application you can query like below to get categories 
http://localhost:8080/api/v1/category
